extensions/proxmox/src/utils/format.ts

```diff
@@ -41,3 +44,18 @@ export function formatStorageSize(size: number): string {
 
   return `${size.toFixed(2)} ${units[unitIndex]}iB`;
 }
+
+export function formatCPU(maxcpu: number): string {
+  return Math.round(maxcpu) + " CPU(s)";
+}
+
+export const formatNumberAsBoolean = (value: number | undefined): string =>
+  typeof value === "number" ? (value === 1 ? "Yes" : "No") : "Unknown";
+export const formatNumberAsSize = (value: number | undefined): string =>
+  typeof value === "number" ? filesize(value) : "Unknown";
```

Comment:
we already have `formatStorageSize`, why bother to add a dependency? is it not enough for some case?